lua/wikis/commons/MatchTicker/DisplayComponents/New.lua

```diff
@@ -141,6 +144,8 @@ function Details:create()
 			:attr('title', 'Add Match Page')
 			:wikitext('+ Add details')
 		), link))
+	elseif self.displayGameIcons then
+		matchBottomBar:node(Game.icon{game=self.match.game, noLink=true, size='50px', spanClass='icon-small'})
```

Comment:
It might be good to introduce a new span class for this, as `icon-small` limits to 25px, which is a tiny bit too large.
`icon-16px` is a bit too small tho. The game icon one in the tournament ticker is limited by `league-icon-small`, which results in 21px